src/class-savvii-dashboard.php

```diff
@@ -162,13 +154,60 @@ function warpdrive_dashboard() {
                     <!-- /Caching defaults -->
             <?php endif; ?>
                     <!-- Read server logs -->
-                    <div class="postbox" style="min-height: 150px;">
+                    <div class="postbox">
                         <h2 class="hndle">Read server logs</h3>
                         <div class="inside"><?= is_super_admin() ? 'Please use the Savvii top menu for reading logs.' : '
                             \'Read server logs\' shows data for all subsites. Because of this, only users with the \'Super Admin\' role are able to read the server logs.'; ?>
                         </div>
                     </div>
                     <!-- /Read server logs -->
+                    <!-- Statistics -->
+                    <div class="postbox">
+                        <h2 class="hndle">Statistics</h2>
+                        <div class="inside">
+                            <div class="pct50">
+                            <?php
+                            $logreader = new ReadLogs();
+                            $errors_502 = $logreader->find_match_in_log( 'access', '/HTTP\/(?:1\.1|2)\"\s+502/i');
+                            $errors_504 = $logreader->find_match_in_log( 'access', '/HTTP\/(?:1\.1|2)\"\s+504/i');
```

Comment:
You're opening and reading the whole access log twice now, is that what you're after? Or could you give multiple regexes? Or make the regexes a bit smarter and do something with the matched values? For example:
```
/HTTP\/(?:1\.1|2)\"\s+(50[24])/i
```
Then you can use the data from the capture group to run a tally on.